tests/e2e/integration/cloud_backup_test.go

```diff
@@ -0,0 +1,214 @@
+package integration_test
+
+import (
+	"bufio"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"github.com/gardener/etcd-backup-restore/pkg/initializer/validator"
+	"github.com/gardener/etcd-backup-restore/pkg/snapstore"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	. "github.com/gardener/etcd-backup-restore/tests/e2e/integration"
+)
+
+var _ = Describe("CloudBackup", func() {
+	var cmdEtcd, cmdEtcdbrctl *Cmd
+	var logger *logrus.Logger
+	var s3Snapstore snapstore.SnapStore
+	var err error
+
+	Describe("Regular backups", func() {
+		BeforeEach(func() {
+			logger := logrus.New()
+			etcdArgs := []string{"--name=etcd",
+				"--advertise-client-urls=http://0.0.0.0:2379",
+				"--listen-client-urls=http://0.0.0.0:2379",
+				"--initial-cluster-state=new",
+				"--initial-cluster-token=new",
+				"--log-output=stdout",
+				"--data-dir=" + os.Getenv("ETCD_DATA_DIR")}
+			cmdEtcd = &Cmd{
+				Task:    "etcd",
+				Flags:   etcdArgs,
+				Logger:  logger,
+				Logfile: filepath.Join(os.Getenv("ETCD_DATA_DIR"), "etcd.log"),
+			}
+			logger.Info("Starting etcd..")
+			go cmdEtcd.RunCmdWithFlags()
+		})
+		BeforeEach(func() {
```

Comment:
Why do we have multiple `BeforeEach`?  Just create a function for each `BeforeEach` callback you created and call those from single before each.